extracting_article_content_from_HTML.py

```diff
@@ -0,0 +1,40 @@
+# Extract article content from HTML with Boilerpipe
+# Be sure to install boilerpipe (pip install boilerpipe)
```

Comment:
Can you explain in the comment above what the input and output of the script is. Its not clear to me.